configuration/configuring-default-applications/en.md

```diff
@@ -3,19 +3,58 @@ title = "Configuring Default Applications"
 +++
 # Configuring Default Applications
 
-You can configure default applications by going to the Budgie Menu, opening Settings app, and going to the Details section.
+Easily change your default web browser, e-mail client, video player, and more by following the steps detailed below.
```

Comment:
Doesn't make much sense to have the "by following the steps detailed below", given this would also be shown on the table / list.